src/SDKs/CognitiveServices/management/CognitiveServices.Tests/CognitiveServices.Tests.csproj

```diff
@@ -10,6 +10,9 @@
   <PropertyGroup>
     <TargetFrameworks>netcoreapp1.1</TargetFrameworks>
   </PropertyGroup>
+  <ItemGroup>
```

Comment:
@felixwa please remove line 13 to 15 and line 35 to 38

Reply:
Remove and also removed AssemblyInfo.cs